packages/pixeloven-storybook/config/package.json

```diff
@@ -26,7 +26,7 @@
     "babel-loader": "8.0.6",
     "css-loader": "3.1.0",
     "deepmerge": "4.0.0",
-    "sass-loader": "7.1.0",
+    "sass-loader": "7.2.0",
```

Comment:
There's some newer releases of sass-loader, but this is the earliest to support dart-sass. I think we could probably go to 7.3.1 safely, but it looks like v8 is breaking changes so not really sure what migration we might need to do if we go that far.